packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -53,6 +53,70 @@ type EdgeLayoutData {
   bendingPoints: [Position!]!
 }
 
+type RectangularNodeCustomizedStyle {
```

Comment:
When I'm reviewing something, the first thing I do is to have a look at any dependency change of GraphQL change to understand the contribution. I've discussed with @frouene on the expected impact on the GraphQL API and I don't understand what I'm seeing right now:

- It has nothing in common with what was discussed
- I don't see how it could work (where is the mutation?)
- It does not match the scope discussed for a first contribution on this topic. I've mentioned it quite explicitly in the comment of the PR of the shape and I've written it myself in the shape